### PR TITLE
feat: add secret for npm auth token

### PR DESCRIPTION
**Pull request by arnikasky**

### CODE REVIEW

**Review by euqen — APPROVED** [review]

File: .github/workflows/publish-apispec.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Publish apispec
 on:
   workflow_call:
     secrets:
-      NPM_AUTH_TOKEN:
+      NPM_TOKEN:
         required: true
```
Comment (fredrikbryntesson) [inline .github/workflows/publish-apispec.yaml:7]: Document in pull request why you in some cases rename NPM_AUTH_TOKEN to NPM_TOKEN.

**Review by andreyvalynko-int — APPROVED** [review]

File: .github/workflows/publish-apispec.yaml [hunk 2]
```diff
@@ -34,14 +34,22 @@ jobs:
 
       - name: Prepare dependencies
         run: |
-          echo ${{ secrets.NPM_AUTH_TOKEN }} > /root/.npmrc
+          echo ${{ secrets.NPM_TOKEN }} > /root/.npmrc
           npm i --quiet @general/apispec-manager@latest
```
Comment (fredrikbryntesson) [inline .github/workflows/publish-apispec.yaml:38]: Do we have any documentation about what is being written from NPM_TOKEN to .npmrc ?
Like is it a only token in the form of like "4hskH7GkdjUJ" or is it concatinated with some other string value?

Comment (arnikasky) [reply .github/workflows/publish-apispec.yaml:38]: https://www.notion.so/intergiro/NPM-Tokens-for-CI-CD-345234f31ab84e26b306c114c56b8f60

File: .github/workflows/publish-apispec.yaml (hunk 1 above)
Comment (arnikasky) [reply .github/workflows/publish-apispec.yaml:7]: https://www.notion.so/intergiro/NPM-Tokens-for-CI-CD-345234f31ab84e26b306c114c56b8f60

**Review by fredrikbryntesson — APPROVED** [review]

**Pull request merged**